thirdparty/faiss/faiss/invlists/InvertedLists.cpp

```diff
@@ -275,19 +276,33 @@ void ArrayInvertedLists::resize(size_t list_no, size_t new_size) {
 
 void ArrayInvertedLists::restore_codes(
```

Comment:
Let's add some comments here. This is a temp code for backward compatibility

Reply:
updated